content/departments/product-engineering/process/planning.md

```diff
@@ -0,0 +1,93 @@
+
+### Planning is iterative
+
+Arriving at a good plan is an iterative process that is neither exclusively top-down nor exclusively bottom-up because there is a cyclical information dependency. Company level strategy is informed by what is working and not working about our business (context our teams have), and team plans are informed by what our company level strategy is (a decision ultimately owned by the CEO). Fundmentally, everyone on our team has context and insights that are valuable inputs into the planning process.
+
+In practice, both the top-down motion and bottoms-up motion happen concurrently, but we need to bootstrap the process to iteratively work through the information dependency. It is helpful to think about planning as having a `W` shape:
+![Planning shape](https://assets.proof.pub/2056/firstround/O6rbG1rNQ2emg8EPj1Xa_TKImage%200%20Leadership%20Frames.png)
+
+Source: [The Secret to a Great Planning Process — Lessons from Airbnb and Eventbrite](https://review.firstround.com/the-secret-to-a-great-planning-process-lessons-from-airbnb-and-eventbrite)
+
+### Building the delivery plan is part of the work
+
+Once a teams has a clear definition of success, there is usually work that needs to be done to figure out what the delivery plan is to achieve that success. The team might need to collect customer feedback, dig in to metrics, or build prototypes as a first step. This exploration is normal, healthy, and part of the delivery plan!
+
+### Plans are ambitious and feasible
+
+We want our plans to be ambitious so that we challenge ourselves to grow. Plans should be ambitious in a focused direction, not ambitious in the number of things we want to accomplish. Doing one thing exceptionally is better than doing multiple things not exceptionally.
+
+Plans need to be feasible. A plan is feasible only if we can achieve it accounting for realistic conditions and expectations (for example: reserve appropriate amount of time for unplanned work). If a plan requires everything to go perfectly, then it is not feasible (because that is not a realistic expectation).
+
+## Process
+
+What follows is our current implementation of the above philosophy.
+
+### Annual planning
+
+The goal of annual planning is to produce the following artifacts:
+
+1. A [good product and engineering strategy](#a-good-strategy-provides-focus) for the year. **[Current product and engineering strategy](../strategy-goals/index.md)**.
+2. A [good plan](#good-plans-are-clear-and-concise) for each team for the year.
+   - A short paragraph about what success looks like for each team at the end of the year. This should include how this success contributes to important top level company needs (as defined in the overall company strategy and product/engineering strategy). This should be documented in the handbook (TBD figure out how ).
```

Comment:
For the TBD, can we use the existing per-team strategy pages? https://handbook.sourcegraph.com/strategy-goals/strategy#per-team-strategy-pages

We could add/modify a standardized section to include this paragraph and link to the project tracker.

Reply:
yeah, I linked to the per team strategy pages for now since that seems like the best place